scripts/lib/zulip_tools.py

```diff
@@ -95,8 +95,11 @@ def run(args, **kwargs):
     try:
         subprocess.check_call(args, **kwargs)
     except subprocess.CalledProcessError:
-        print(WARNING + "A command run by %s failed; see output above." % (sys.argv[0])
+        print ("\n")
+        print('\x1b[0;30;43m' + "A command run by %s failed; see output above." % (sys.argv[0])
               + ENDC)
-        print(WARNING + "Command was: %s" % (" ".join(args))
+        print ("\n")
+        print('\x1b[0;37;41m' + "Command that failed was: %s" % (" ".join(args))
```

Comment:
It's defined at the top of this file.  But you shouldn't edit that color -- if you want another color, add a new one next to WARNING.